python/cuspatial/cuspatial/_lib/spatial_window.pxd

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+
+from cudf._lib.legacy.cudf cimport *
+
+from cudf._lib.column cimport column, column_view
+from cudf._lib.table cimport table, table_view
+from cudf._lib.move cimport move, unique_ptr
+
+cdef extern from "spatial_window.hpp" namespace "cuspatial" nogil:
+    cdef unique_ptr[table] cpp_points_in_spatial_window \
```

Comment:
In libcudf we made the decision to keep the `pxd` files in a separate directory (`_lib/cpp/`) from the `pyx` files because files within the same directory end up "smashed together" in the same "namespace". This was done so we could remove the `cpp_` prefix without conflict. Dependent files could then explicitly `import ... as ...` in the usual manner.

```suggestion
    cdef unique_ptr[table] points_in_spatial_window \
```